jujugui/static/gui/src/app/components/boolean-config/_boolean-config.scss

```diff
@@ -4,13 +4,18 @@
     position: relative;
 
     &--title {
-      padding-top: 8px;
+        margin: auto;
+        margin-left: 0;
     }
 
     &--toggle {
-      position:absolute;
-      right: 0;
-      top: 0;
+        margin: auto;
+        margin-left: 4px;
+        margin-right: 0;
+
+        &-container {
```

Comment:
`container--toggle` makes it seem like it's some control that toggles a container. It isn't; it's just a container to make laying out the toggle widget and its label easier, hence `toggle--container`.